syntax = "proto3";

option java_multiple_files = true;
option java_package = "ex.grpc";

package mathtest;

service MT {
    rpc getValue (MathRequest) returns (MathReply) {}
}

message MathRequest {
    int32 a = 1;
    int32 b = 2;
}

message MathReply {
    int32 result = 1;
}

